.travis.yml

```diff
@@ -41,3 +41,7 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+notifications:
+  slack:
+    secure: $SLACK_NOTIFICATION_KEY
```

Comment:
To be consistent with what we used to do with IRC, probably want
```
    on_success: change
```
in here so we don't see messages for all the successful builds, or maybe we do?

Reply:
Done